docs/github-labels.md

```diff
@@ -0,0 +1,117 @@
+| lifecycle/frozen                   | Indicates that an issue or PR should not be auto-closed due to staleness. | Any |
+| lifecycle/rotten                   | Denotes an issue or PR that has aged beyond stale and will be auto-closed. | Any |
+| lifecycle/stale                    | Denotes an issue or PR has remained open with no activity and has become stale. | Any |
+| priority/awaiting-more-evidence    | Lowest priority. Possibly useful, but not yet enough support to actually get it done. | Any |
+| priority/backlog                   | Higher priority than priority/awaiting-more-evidence. | Any |
+| priority/critical-urgent           | Highest priority. Must be actively worked on as someone's top priority right now. | Any |
+| priority/important-longterm        | Important over the long term, but may not be staffed and/or may need multiple releases to complete. | Any |
+| priority/import-soon               | Must be staffed and worked on either currently, or very soon, ideally in time for the next release. | Any |
+| ready-to-work                      | Indicates that an issue or PR has been sufficiently triaged and prioritized and is now ready to work. | Any |
+| size/L                             | Denotes a PR that changes 100-499 lines, ignoring generated files. | Any |
+| size/M                             | Denotes a PR that changes 30-99 lines, ignoring generated files.| Any |
+| size/S                             | Denotes a PR that changes 10-29 lines, ignoring generated files.| Any |
+| size/XL                            | Denotes a PR that changes 500+ lines, ignoring generated files.| Any |
+| size/XS                            | Denotes a PR that changes 0-9 lines, ignoring generated files.| Any |
+| triage/duplicate                   | Indicates an issue is a duplicate of other open issue. | Humans |
+| triage/needs-information           | Indicates an issue needs more information in order to work on it. | Humans |
+| triage/not-reproducible            | Indicates an issue can not be reproduced as described. | Humans |
+| triage/unresolved                  | Indicates an issue that can not or will not be resolved. | Humans |
+
+## Labels that apply only to issues
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| good first issue                   | Denotes an issue ready for a new contributor, according to the "help wanted" guidelines. | Anyone |
+| help wanted                        | Denotes an issue that needs help from a contributor. Must meet "help wanted" guidelines. | Anyone |
+
+
+## Labels that apply only to PRs
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| approved                           | Indicates a PR has been approved by OWNERS in accordance with [GOVERNANCE.md](GOVERNANCE.md) guidelines. | Maintainers |
+| vmware-cla: no                     | Indicates the PR's author has not signed the (VMware CLA)[https://cla.vmware.com/faq] | VMware CLA Bot |
```

Comment:
The link does not render properly: there can be no whitespace between `(VMware CLA)` and `[https://cla.vmware.com/faq]`

Reply:
Corrected.

Reply:
still not fixed (when I look at the Github markdown rendering for the file), maybe they need to be on the same line...